spec/support/medium_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+require 'pry'
+
+RSpec.shared_examples "media" do
+  describe "controller validations" do
+    describe "GET 'index'" do
+      it "is successful" do
+        get :index
+        expect(response).to have_http_status 200
```

Comment:
Should we expect this to render the `:index` template, as we do with the other GET actions?